parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -51,17 +51,6 @@ retrycmd_if_failure() {
   done
   echo Executed \"$@\" $i times
 }
-retrycmd_if_failure_no_stats() {
```

Comment:
It's not clear to me why we still have this option to not report retries in stdout. I think this may be leftover from a time when apt operations were using this func, and stdout/stderr requirements were weird. Dunno.

In any event I think we should just use one, and I think the retry echo statements are valuable for quickly debugging CSE output.

Reply:
nevermind it's because apt